### ISSUE TITLE
Cant import `bootstrapValidate` function using ES6/Webpack

### ISSUE THREAD

**Issue by babakgh** [PascaleBeier/bootstrap-validate#21]
I got this error `Uncaught TypeError:__WEBPACK_IMPORTED_MODULE_2_bootstrap_validate__ is not a function`
via code below
```
import * as bootstrapValidate from 'bootstrap-validate'

bootstrapValidate('#test', 'required')
```

ps: this is obviously wrong syntax `import * from 'bootstrap-validate' as bootstrapValidate;`
https://bootstrap-validate.js.org/installation.html

**Comment by babakgh** [comment]
my `package.json`

```
  "dependencies": {
    "@rails/webpacker": "^3.4.1",
    "bootstrap": "^4.1.0",
    "bootstrap-validate": "2.0.0",
    "jquery": "^3.3.1",
    "popper.js": "^1.14.1",
    "rails-ujs": "^5.1.5",
    "sweetalert2": "^7.18.0"
  },
```

**Comment by PascaleBeier** [comment]
Yeah, this is a typo.

Could you provide your webpack configuration and version?

**Comment by PascaleBeier** [comment]
Fixed in da4af447a6e0267fee107e9a154ca207a6cb94bd
